packages/ai/mcp-stdio/get-environment.ts

```diff
@@ -0,0 +1,37 @@
+export function getEnv(
+  customEnv?: Record<string, string>,
```

Comment:
wondering how to rename this to make it more clear? also, it might be worth a comment mentioning if it's defaults or overrides.